src/main/java/de/caritas/cob/agencyservice/api/service/AgencyService.java

```diff
@@ -145,6 +144,27 @@ public List<FullAgencyResponseDTO> getAgencies(Optional<String> postCode,
     return mutableResponseDTO;
   }
 
+  /**
+   * Returns a list of {@link AgencyResponseDTO} from the current tenant, which match to the provided
```

Comment:
remove redundant comment, as we are trying to follow cleanCode, getAgencies is self-explanatory